book/src/ui-usage.md

```diff
@@ -1,73 +1,105 @@
 # Usage
+Siren offers many features ranging from diagnostics, logs, validator management including graffiti and exiting. Below we will describe all major features and how to take advantage of Siren to the fullest.
 
-## Dashboard
+# Dashboard
 
 Siren's dashboard view provides a summary of all performance and key validator metrics. Sync statuses, uptimes, accumulated rewards, hardware and network metrics are all consolidated on the dashboard for evaluation.
 
 ![dashboard](imgs/ui-dashboard.png)
 
-## Account Earnings
+### Account Earnings
 
 The account earnings component accumulates reward data from all registered validators providing a summation of total rewards earned while staking. Given current conversion rates, this component also converts your balance into your selected fiat currency.
 
 Below in the earning section, you can also view your total earnings or click the adjacent buttons to view your estimated earnings given a specific time frame based on current device and network conditions.
 
+Keep in mind, if validators have activated withdrawals this balance will only reflect the balance before the deposit is made and will subsequently be reset to a zero balance until the following deposit period.
```

Comment:
I am not following on this. Usually there is only 1 deposit made to the validator - that's the 32 ETH deposit to activate the validators. So I don't get it about:
- "before the deposit is made"
- "until the following deposit period" - does it mean to be "withdrawals period"? 

The term "activated withdrawals" could also be replaced with "updated withdrawal credentials".

Reply:
i see how the wording could confuse. Im referring to the rewards payout, so i named it such now